packages/f/freefilesync/package.yml

```diff
@@ -1,38 +1,40 @@
 name       : freefilesync
-version    : 10.3
-release    : 4
+version    : '13.1'
+release    : 5
 source     :
-    - https://freefilesync.org/download/FreeFileSync_10.3_Source.zip : 9e958ae968d4e8aff40e547edc575735da73fe0dd9d2558b060f11471a24a0ef
+    - https://freefilesync.org/download/FreeFileSync_13.1_Source.zip : c6e25039e8266473ec0a6254d9a9130f89d0874e4cfaba678d05447abdf74829
 license    : GPL-3.0-or-later
 component  : network.util
 summary    : Free backup software to synchronize files and folders
 description: |
     FreeFileSync is a folder comparison and synchronization software that creates and manages backup copies of all your important files.
 builddeps  :
-    - pkgconfig(gtk+-2.0)
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(libcurl)
     - pkgconfig(libnotify)
+    - pkgconfig(libssh2)
     - pkgconfig(sm)
     - pkgconfig(xxf86vm)
     - libboost-devel
-    - wxwidgets30-devel
+    - wxwidgets-devel
 setup      : |
     pushd ..
     %apply_patches
     popd
 build      : |
-    %make -C Source
-    %make -C Source/RealTimeSync
+    %make -C Source exeName=FreeFileSync
+    %make -C Source/RealTimeSync exeName=RealTimeSync
 install    : |
-    %make_install -C Source
-    %make_install -C Source/RealTimeSync
+    #%make_install -C Source
+    #%make_install -C Source/RealTimeSync
 
+    install -Dm00755 $workdir/Build/Bin/FreeFileSync $installdir/usr/bin/FreeFileSync
+    install -Dm00755 $workdir/Build/Bin/RealTimeSync $installdir/usr/bin/RealTimeSync
     install -Dm00644 $pkgfiles/FreeFileSync.desktop $installdir/usr/share/applications/FreeFileSync.desktop
     install -Dm00644 $pkgfiles/RealTimeSync.desktop $installdir/usr/share/applications/RealTimeSync.desktop
 
-    unzip Build/Resources.zip FreeFileSync.png RealTimeSync.png
-    install -Dm00644 FreeFileSync.png $installdir/usr/share/pixmaps/FreeFileSync.png
-    install -Dm00644 RealTimeSync.png $installdir/usr/share/pixmaps/RealTimeSync.png
-
-    # Set file permissions
-    find $installdir/usr/share/FreeFileSync -type f -print0 | xargs -0 chmod 644
-    find $installdir/usr/share/doc/FreeFileSync -type f -print0 | xargs -0 chmod 644
+    #unzip Build/Resources.zip FreeFileSync.png RealTimeSync.png
+    mkdir $installdir/usr/share/FreeFileSync
+    install -Dm00644 $workdir/Build/Resources/* $installdir/usr/share/FreeFileSync
+    install -Dm00644 $pkgfiles/FreeFileSync.png $installdir/usr/share/pixmaps/FreeFileSync.png
+    install -Dm00644 $pkgfiles/RealTimeSync.png $installdir/usr/share/pixmaps/RealTimeSync.png
```

Comment:
Also curious about this set of changes.

Reply:
There is no install routine in the makefile and a bunch of other things changed. I'm still looking at other distro's packaging for this version series.